product_docs/docs/biganimal/release/administering_cluster/projects.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Managing projects
+description: "Use projects to provide additional security and segregation to workflows within your organization"
+---
+
+Projects are a group of database clusters within a BigAnimal organization. Access to projects is controlled by a user's role. With projects, you can separate workflows to provide secure and isolated environments and assign different users to different projects. See [Managing portal access](/biganimal/release/administering_cluster/01_portal_access) for more information.
+
+Projects actions are available from the left navigation bar. The actions apply to the project in focus. You can switch between projects using the Projects menu.
+
+## Adding a user to a project
+Before adding a user to a project: 
+- The user must be added through your organization's identify provider.
```

Comment:
can this be reworded to "The user must be added to the organization through the organization's identity provider"